src/components/ebay-phone-input/component.ts

```diff
@@ -0,0 +1,132 @@
+import type { WithNormalizedProps } from "../../global";
+import type { Input as MenuButtonInput } from "../ebay-menu-button/component";
+import type { default as Textbox } from "../ebay-textbox/component-browser";
+import mask, { stripNonDigits } from "../../common/mask";
+
+import countries, {
+    type CountriesLanguageInterface,
+    type CountryInterface,
+    countriesEnglish,
+} from "../../common/countries";
+import { AttrString } from "marko/tags-html";
+
+export interface CountryNames {
+    [index: string]: string;
+}
+
+export interface PhoneInputEvent {
+    originalEvent?: Event;
+    value?: string;
+    rawValue?: string;
+    callingCode?: string;
+    countryCode?: string;
+}
+
+interface PhoneInputInput extends Omit<MenuButtonInput, `on${string}`> {
+    disabled?: boolean;
+    "country-names"?: CountriesLanguageInterface;
+    "a11y-button-text"?: AttrString;
```

Comment:
What are a11y-button-text and a11y-menu-text used for? I don't see them explained in storybook docs. We might need more descriptive names for them also. Is a11y-button-text the prefix for country code listbox button? e.g. "Country: "?

Reply:
Done, fixed the documentation fort his.